lms/djangoapps/api_manager/receivers.py

```diff
@@ -1,11 +1,12 @@
 """
-Signal handlers supporting various gradebook use cases
+Signal handlers supporting various gradebook, course metadata use cases
```

Comment:
We should remove "gradebook" from this comment -- there's no gradebook integration here, which is confusing